src/OrbitGl/TimerTrack.h

```diff
@@ -84,6 +84,13 @@ class TimerTrack : public Track {
     return true;
   }
 
+  void DrawTimer(TextBox* prev_text_box, TextBox* current_text_box, TextBox* next_text_box,
```

Comment:
I'm not finding any recommendation on limits in the arguments of a function, but seems to be very big. If you want to make a function here (which makes sense), don't you think that we would want to have 2 or 3 different functions here?

Reply:
Well in the end, there would be one function surrounding which needs all the arguments, so I guess I'd keep it that way.

Reply:
Since this is probably the code that runs the most in Orbit, there are also performance implications in passing all those arguments. The first six are passed through registers on Linux, but the rest is constantly being pushed and popped from the stack. It could make sense to bundle invariants in single struct that we would pass be reference to the DrawTimer function.

Reply:
Maybe adding `inline` annotation would help in this case. I know its not bullet proof, though.

Reply:
I agree with Pierric here, after the PR we should find a better way to organize the parameters.

Reply:
For performance: Make sure you measure, it's fairly straightforward with the introspection window.